api/src/wfl/module/covid.clj

```diff
@@ -0,0 +1,41 @@
+(ns wfl.module.covid
+  "Handle COVID processing."
+  (:require [clojure.data.json :as json]
+            [clojure.set :as set]
+            [clojure.string :as str]
+            [clojure.tools.logging.readable :as log]
+            [wfl.api.workloads :as workloads :refer [defoverload]]
+            [wfl.jdbc :as jdbc]
+            [wfl.module.batch :as batch]
+            [wfl.service.datarepo :as datarepo]
+            [wfl.service.firecloud :as firecloud]
+            [wfl.service.google.storage :as gcs]
+            [wfl.service.postgres :as postgres]
+            [wfl.util :as util]
+            [wfl.wfl :as wfl]))
+
+(def pipeline "Sarscov2IlluminaFull")
+
+;; TODO: implement COVID workload creation
+;;  - make sure permissions/inputs are right upfront
+;;  - dispatch on source/sink/executor
+;;  - store information into database
+(defn create-covid-workload!
+  [tx {:keys [source sink executor] :as request}])
+
+;; TODO: implement progressive (private) functions inside this update loop
```

Comment:
Thanks for flagging this.  Looks like this is the space in which we'll all be working simultaneously.

Reply:
Yup! That's what I'm imagining, we could write function outside of this and thread them in this block with `cond` or `when`s.